http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpDockerDAO.scala

```diff
@@ -192,6 +196,10 @@ object HttpDockerDAO {
     } yield ManifestConfig(mediaType, size, digest)
   }
 
+  implicit val envDecoder: Decoder[Env] = Decoder.decodeString.map { s =>
```

Comment:
Seems this could fail if the string is not in the format `A=B`? Perhaps this should be `emap`?